bases/ecoindex/cli/arguments_handler.py

```diff
@@ -36,8 +38,13 @@ def get_urls_from_file(urls_file: FilePath) -> Set[str]:
 
 def get_urls_recursive(main_url: str) -> Set[str]:
     parsed_url = urlparse(main_url)
-    domain = parsed_url.netloc
-    main_url = f"{parsed_url.scheme}://{domain}"
+    netloc = parsed_url.netloc
+    domain = netloc
+    if Settings().DOCKER_CONTAINER:
+        if (parsed_url.hostname == "localhost"):
```

Comment:
**🎯 suggestion(non-blocking):** Here, the more "pythonic" approach is something like that

```suggestion
    if Settings().DOCKER_CONTAINER and parsed_url.hostname == "localhost":
```